lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.call(alderaan).bind(alderaan);
```

Comment:
you only need a `.call` here. Read the link i posted in the `links` slack channel